TestScene.gd

```diff
@@ -5,13 +5,16 @@ extends CenterContainer
 @export var entity:PandoraEntity
 
 
+@onready var label = $Label
+
+
 func _ready() -> void:
-	print(item.get_entity_id(), " - ", item.get_entity_name())
+	label.text += item.get_entity_id() + " - "+ item.get_entity_name() + '\n'
```

Comment:
This now shows correctly the loaded entities from a compressed file in release build